Sources/Penny/ProposalsChecker.swift

```diff
@@ -348,8 +353,8 @@ private extension Proposal.User {
     }
 }
 
-private struct QueuedProposal {
-    let uuid = UUID()
+struct QueuedProposal: Codable {
+    var uuid = UUID()
```

Comment:
Defaulted values and `Codable` are generally a bad mix, even if they're mutable.